README.md

```diff
@@ -11,6 +11,32 @@ Docker and the aws-cli programs to run locally.
 [Get Docker](https://docs.docker.com/get-docker/)
 [Get aws-cli](https://docs.aws.amazon.com/cli/latest/userguide/getting-started-install.html)
 
+
+## Building and Running Your Own H3lioviz-Server for swxtrec-cdk
+### Setup h3lioviz-server for Building
+
+1. Navigate to https://www.paraview.org/download/ and download *ParaView-5.10.1-osmesa-MPI-Linux-Python3.9-x86_64.tar.gz* which will actually download as *ParaView-5.10.1-osmesa-MPI-Linux-Python3.9-x86_64.tar*
```

Comment:
I commented on #59  that this wasn't the case for me. You might need to download the Headless version.